pkg/volume/gce_pd/attacher.go

```diff
@@ -165,6 +165,9 @@ func (attacher *gcePersistentDiskAttacher) MountDevice(spec *volume.Spec, device
 	options := []string{}
 	if readOnly {
 		options = append(options, "ro")
+	} else {
+		// as per https://cloud.google.com/compute/docs/disks/add-persistent-disk#formatting
```

Comment:
Good point.  This just got complicated.

On Wed, Jul 6, 2016 at 11:51 PM, Michael Rubin <notifications@github.com>
wrote:

> In pkg/volume/gce_pd/attacher.go
> <https://github.com/kubernetes/kubernetes/pull/28448#discussion_r69858276>
> :
>
> > @@ -165,6 +165,9 @@ func (attacher *gcePersistentDiskAttacher) MountDevice(spec *volume.Spec, device
> >  	options := []string{}
> >  	if readOnly {
> >  		options = append(options, "ro")
> > +	} else {
> > +		// as per https://cloud.google.com/compute/docs/disks/add-persistent-disk#formatting
>
> We need to test for ext4 also. I don't think ext2 (nor implictly ext3
> supports this mount option)?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/28448/files/8efefab9a3fdc4906ca53d4a23d22c3846d0610b#r69858276>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AFVgVG1VbgifAwcL_sXxrRhsOd1YPG9Dks5qTKIEgaJpZM4JECrD>
> .
>